script.js

```diff
@@ -8,17 +8,17 @@ function changeColor()
 
     let input = prompt('Input text'); //a variable that stores user input
     let wordCount, letterCount, sentenceCount;  // variables for output
-    if(input!==null)
+    if(input!==null) //an exception if user cancels input
     {
         document.getElementById("cancelText").style.display="none";
     }
-    else
+    else // the other case
     {
         alert('You hit the cancel button!');
         document.getElementById("cancelText").style.display="block";
 
     }
-    function countLetters(text)
+    function countLetters(text) // function that counts letters in text
```

Comment:
Your counter counts numbers as letters , your function has to have a parameter to count only letters from A-Z